.github/workflows/build-and-publish-base.yaml

```diff
@@ -52,7 +53,8 @@ jobs:
         DEFAULT_BRANCH: "plugins"
         DOCKER_USER: ${{ secrets.DOCKERHUB_USERNAME}}
         DOCKER_PASS: ${{ secrets.DOCKERHUB_PASSWORD}}
-        IMAGE_TAG: ${{ env.BASE_TAG}}
+        # On merge to default 'main' branch push current 'dev-base' Docker tag
+        IMAGE_TAG: dev-base
```

Comment:
Why did we hardcode this value here?  Before it was using the env.BASE_TAG value

Reply:
Hardcoding is needed here as it's a special case for the `main` branch where this custom naming applies.

Reply:
I see. It's also used in the same workflow below: 
https://github.com/bitovi/bitops/blob/c10fb076f88677a0d299f39be0775dd910b79fd9/.github/workflows/build-and-publish-base.yaml#L113

Reply:
I think that answered the main concern now.